jenkins/requirements.txt

```diff
@@ -1 +0,0 @@
-requests==2.5.3
```

Comment:
Hey @aleffert we are using jenkins/requirements.txt, but updating that would not be a big deal. We'd just want to do it at the right time (i.e. when you merge this code). 

For purposes of completion, here's what we have:
```bash
$ grep "pip install" */config.xml
build-jenkins-worker-ami/config.xml:#pip install -U ansible==1.8
clean-up-AMIs/config.xml:pip install -r aws/requirements.txt
microsites-staging-create-baseline/config.xml:pip install -r requirements.txt
microsites-staging-create-baseline/config.xml:#pip install selenium==2.42.1
testeng-ci-deduper/config.xml:pip install -q -r jenkins/requirements.txt
testeng-ci-deduper/config.xml:pip install -q -r aws/requirements.txt
```